.github/CHANGELOG.md

```diff
@@ -1,105 +1,188 @@
+  ```
+
+  For more details, see the
+  [Xanadu cloud platform](https://strawberryfields.readthedocs.io/en/stable/introduction/remote.html)
+  tutorial.
+
+* Significantly speeds up the Fock backend of Strawberry Fields,
+  through a variety of changes:
+
+  - The Fock backend now uses The Walrus high performance implementations of
+    the displacement, squeezing, two-mode squeezing, and beamsplitter operations.
+    [(#287)](https://github.com/XanaduAI/strawberryfields/pull/287)
+    [(#289)](https://github.com/XanaduAI/strawberryfields/pull/289)
+
+  - Custom tensor contractions which make use of symmetry relations for the beamsplitter
+    and the two-mode squeeze gate have been added, as well as more efficient contractions
+    for diagonal operations in the Fock basis.
+    [(#292)](https://github.com/XanaduAI/strawberryfields/pull/292)
+
+<br>
+
+* New `sf` command line program for configuring Strawberry Fields for access
+  with the Xanadu cloud platform, as well as submitting and executing jobs from the
+  command line.
+  [(#146)](https://github.com/XanaduAI/strawberryfields/pull/146)
+  [(#312)](https://github.com/XanaduAI/strawberryfields/pull/312)
+
+  The new Strawberry Fields command line program `sf` provides several utilities
+  including:
+
+  * `sf configure [--token] [--local]`: configure the connection to the cloud platform
+
+  * `sf run input [--output FILE]`: submit and execute quantum programs from the command line
+
+  * `sf --ping`: verify your connection to the Xanadu cloud platform.
+
+  For more details, see the
+  [documentation](https://strawberryfields.readthedocs.io/en/stable/code/sf_cli.html).
+
+* New configuration functions to load configuration from
+  keyword arguments, environment variables, and configuration files.
+  [(#298)](https://github.com/XanaduAI/strawberryfields/pull/298)
+  [(#306)](https://github.com/XanaduAI/strawberryfields/pull/306)
+
+  This includes the ability to automatically store Xanadu cloud platform
+  credentials in a configuration file using the new function
+
+  ```python
+  sf.store_account("AUTHENTICATION_TOKEN")
+  ```
+
+  as well as from the command line,
+
+  ```bash
+  $ sf configure --token AUTHENTICATION_TOKEN
+  ```
+
+  Configuration files can be saved globally, or locally on a per-project basis.
+  For more details, see the
+  [configuration documentation](https://strawberryfields.readthedocs.io/en/stable/introduction/configuration.html)
 
 * Adds the `x_quad_values` and `p_quad_values` methods to the `state` class.
   This allows calculation of x and p quadrature
   probability distributions by integrating across the Wigner function.
-  [#270](https://github.com/XanaduAI/strawberryfields/pull/270)
+  [(#270)](https://github.com/XanaduAI/strawberryfields/pull/270)
 
 * Adds support in the applications layer for node-weighted graphs. Users can sample from graphs
   with node weights using the WAW encoding and input node weights into search algorithms in the
```

Comment:
Users might not know what the WAW encoding is

Reply:
I suggest we address this is a new smaller PR after `demo` is merged, as it is a pre-existing changelog entry.

Reply:
@trbromley

Reply:
:thinking: Currently the user could look at the linked PRs to find out
To include it here, are we able to have math equations in markdown?
(but yes I agree this could be done outside of this PR)

Reply:
Easiest thing is to not give any specifics that we have to define. Maybe "sample from graphs with node weights using special-purpose encodings"